server/controllers/TransactionController.js

```diff
@@ -21,6 +22,24 @@ class TransactionController {
         error: 'Account does not exist',
       });
     }
+
+  
+    let transporter = nodemailer.createTransport({
+      host: "smtp.gmail.com",
+      port: 587,
+      secure: false,
+      auth: {
+        user: 'igwechinonso77@gmail.com', 
+        pass: '081658424422'
+      }
+    });
+    await transporter.sendMail({
+      from: '"MyBanka" igwechinonso77@gmail.com',
+      to: "igwechinonso1994@gmail.com",
+      subject: "Debit Notification",
+      html: `Your account was debited by ${request.body.amount}`
+    });
+
     const transaction = await Transaction.debit(request.body);
     return response.status(200).json({
       status: 200,
```

Comment:
is there a reason you are using 200 status code with an error message on the next line?